docs/track.rst

```diff
@@ -2964,6 +2964,41 @@ The following meta data is always returned:
 * ``weight``: The number of fetched pages. Should always equal to the ``pages`` parameter.
 * ``unit``: The unit in which to interpret ``weight``. Always "ops".
 
+field-caps
+~~~~~~~~~~~~~~~~~~~
+
+Retrieve `the capabilities of fields among indices. <https://www.elastic.co/guide/en/elasticsearch/reference/current/search-field-caps.html>`_.
+
+Properties
+""""""""""
+
+* ``index`` (optional, defaults to `_all`): An `index pattern <https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-index.html>`_ that defines which indices should be targeted by this operation.
```

Comment:
```suggestion
* ``index`` (optional, defaults to ``_all``): An `index pattern <https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-index.html>`_ that defines which indices should be targeted by this operation.
```

Have you considered linking to https://www.elastic.co/guide/en/elasticsearch/reference/current/api-conventions.html#api-multi-index instead? Landing on a page with only two links is a bit unsettling, but it may be the best we have.

Reply:
Good suggestion. I just copied this from other runners.

Reply:
I've reverted to using the previous link to be consistent with other runners. If we decide to change this link, I think we need to fix other links too.